utils/imgutils/gif.go

```diff
@@ -418,12 +398,14 @@ func (d *decoder) readImageDescriptor() error {
 	br := &blockReader{d: d}
 	lzwr := lzw.NewReader(br, lzw.LSB, int(litWidth))
 	defer lzwr.Close()
-	if err = readFull(lzwr, m.Pix); err != nil {
+
+	if _, err := io.Copy(ioutil.Discard, lzwr); err != nil {
```

Comment:
`ioutil.Discard` is now in `io.Discard`. The `ioutil` package is no longer preferred.

Reply:
Good point, I'll update.